db/airports.json

```diff
@@ -0,0 +1,122 @@
+		"city": "Kansas City",
+		"state": "MO",
+		"country": "US",
+		"airportCode": "MCI",
+    "isInternational": false
+	},
+	"airport2": {
+		"uid": "Stephen",
+		"airportName": "Dubai International",
+		"imgUrl": "https://gapyearsite.wpengine.com/wp-content/uploads/2017/11/articles/DubaiNight1.jpg",
+		"city": "Dubai",
+		"state": "N/A",
+		"country": "UAE",
+		"airportCode": "DXB",
+    "isInternational": true
+	},
+	"airport3": {
+		"uid": "Kenny",
+		"airportName": "John F. Kennedy Airport",
+		"imgUrl": "https://cdn11.bigcommerce.com/s-45hj43/images/stencil/500x659/products/4559/6793/CW_808_Statue_of_Liberty_with_New_York_Skyline__35635.1557789140.jpg?c=2&imbypass=on",
+		"city": "New York City",
+		"state": "NY",
+		"country": "US",
+		"airport code": "JFK",
+		"isInternational": true
+	},
+	"airport4": {
+		"uid": "Zoe",
+		"airportName": "Honolulu International Airport",
+		"imgUrl": "https://www.nationalgeographic.com/content/dam/travel/2018-digital/city-guides-2018/honolulu-hawaii/waikki-beach-honolulu-hawaii.jpg",
+		"city": "Honolulu",
+		"state": "Hawaii",
+		"country": "US",
+		"airport code": "HNL",
+		"isInternational": false
+	},
+	"airport5": {
+		"uid": "Mary",
+		"airportName": "Edinburgh Airport",
+		"imgUrl": "https://cdn-image.departures.com/sites/default/files/1549985842/west-bow-street-architecture-edinburgh-scotland-EDINBURGH0219.jpg",
+		"city": "Edinburgh",
+		"state": "N/A",
+		"country": "SCT",
+		"airport code": "EDI",
+		"isInternational": true
+	},
+	"airport6": {
+		"uid": "Sarah",
+    "airportName": "Nashville International Airport",
+    "imgUrl": "https://render.fineartamerica.com/images/rendered/default/greeting-card/images/artworkimages/medium/1/nashville-skyline-at-dusk-2018-panorama-color-jon-holiday.jpg?&targetx=-400&targety=0&imagewidth=1500&imageheight=500&modelwidth=700&modelheight=500&backgroundcolor=AD7296&orientation=0",
+		"city": "Nashville",
+		"state": "TN",
+		"country": "US",
+		"airportCode": "BNA",
+		"isInternational": false
+	},
+	"airport7": {
+		"uid": "Stephen",
+		"airportName": "Sydney Airport",
+		"imgUrl": "https://i.pinimg.com/originals/25/04/f6/2504f628bdb2090690c220e14b341717.jpg",
+		"city": "Sydney",
+		"state": "N/A",
+		"country": "AU",
+		"airportCode": "SYD",
+		"isInternational": true
+	},
+	"airport8": {
+		"uid": "Kenny",
+		"airportName": "O’Hare International Airport",
+		"imgUrl": "https://images.discerningassets.com/image/upload/q_auto:best/c_limit,h_1000,w_1000/v1564957074/One_Night_in_Chicago_sm_vrx6cr.jpg",
+		"city": "Chicago",
+		"state": "IL",
+		"country": "US",
+		"airport code": "ORD",
+		"isInternational": false
+	},
+	"airport9": {
+		"uid": "Zoe",
+		"airportName": "Leonardo da Vinci International Airport Rome",
+		"imgUrl": "https://www.fodors.com/wp-content/uploads/2018/10/HERO_UltimateRome_Hero_shutterstock789412159.jpg",
+		"city": "Rome",
+		"state": "N/A",
+		"country": "ITL",
+		"airport code": "FCO",
```

Comment:
airportCode

Reply:
Maybe I missed one of them... And you can reject my PR again